scripts/npm.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Pass arguments to npm the project root
+
+set -e
+set -x
+
+ARGS=$*
+
+vagrant ssh -c "cd /vagrant/src/app/ && npm $ARGS"
```

Comment:
Is this script necessary any longer, considering that we now have `update` and `server`?